api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -458,6 +458,7 @@ def patch_event_date(
                 beginning_datetime=update_body.get("beginning_datetime", offers_api.UNCHANGED),
                 editing_provider=current_api_key.provider,
             )
+        offers_api.handle_stocks_edition([(stock_to_edit, body.beginning_datetime != None, False)])
```

Comment:
C’est du nitpicking mais sans aller voir la définition de la méthode on ne comprend vraiment pas ce que l’on passe et pourquoi, est-ce que c’est envisageable de faire quelque chose comme ça ?

```suggestion
        is_beginning_date_updated = body.beginning_datetime != None
        should_booking_canceled = False
        offers_api.handle_stocks_edition([(stock_to_edit, is_beginning_date_updated, should_booking_canceled)])
```

Reply:
même mieux en faite

```suggestion
            edited_date, is_beginning_updated = offers_api.edit_stock(
                stock_to_edit,
                quantity=quantity + stock_to_edit.dnBookedQuantity if isinstance(quantity, int) else quantity,
                price_category=price_category,
                booking_limit_datetime=update_body.get("booking_limit_datetime", offers_api.UNCHANGED),
                beginning_datetime=update_body.get("beginning_datetime", offers_api.UNCHANGED),
                editing_provider=current_api_key.provider,
            )
        should_booking_canceled = False
        offers_api.handle_stocks_edition([(stock_to_edit, is_beginning_updated, should_booking_canceled)])
```

Reply:
bien vu, merci pour la reco